homework/module_1-hw_standardlib/task_2.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+
+def check_ip(ip_address) -> bool:
+    """This function validates the given ip address and returns True if it's valid IPv4, otherwise False"""
+    is_valid = True
+    ip_address_pattern_match = re.match(
```

Comment:
We can immediately return False if the pattern does not match.

Reply:
fixed